mwdb/web/src/components/RichAttribute/MarkedMustache.tsx

```diff
@@ -112,7 +118,12 @@ class MustacheContext extends Mustache.Context {
         }
         if (!name) return undefined;
         const path = splitName(name);
-        let currentObject = this.view;
+        // In case of lambdas, the subrenderer makes
+        // this.view be a MustacheContext so make sure
+        // we get the actual view
+        let currentObject = this.view instanceof MustacheContext
+                                ? this.view.view
+                                : this.view;
```

Comment:
That's caused by incorrect overload of `Mustache.Writer.render`, I didn't know that it's called recursively (https://github.com/janl/mustache.js/blob/972fd2b27a036888acfcb60d6119317744fac7ee/mustache.js#L593). 

I think the best solution is to create root MustacheContext in renderValue instead of overloading `render`